etcdserver/etcdserverpb/rpc.proto

```diff
@@ -763,6 +763,8 @@ message MemberListResponse {
   ResponseHeader header = 1;
   // members is a list of all members associated with the cluster.
   repeated Member members = 2;
+  // proxies is a list of all proxy members associated with the cluster.
+  repeated Member proxies = 3;
```

Comment:
Hold off on having an explicit `proxies` list until it's necessary? Can just clobber the `members` list with proxy data for the time being. That way old clients autosyncing on a proxy won't sync to the core cluster.